### PR TITLE
Typoci

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- closes https://github.com/RasaHQ/rasa/issues/5720
- Add TypoCI configuration to check docs
- fix legitimate typos found

See https://github.com/melindaloubser1/rasa/pull/3 for behaviour on docs

Things learned about Typo CI:
- only checks changed lines, so in order to test what it will flag the above PR changed every line of the docs. 
- capitalization at the beginning of a word doesn't seem to matter, but mid-word capitalization does
- if one part of a camel cased or hyphenated word is flagged, you only need to add the offending part to the ignore words list to get past it
- For compound words with no boundary markers, you have to add the whole thing, even if all the parts of the word are already ignored.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by melindaloubser1** [comment]
@akelad the [typo ci](https://github.com/marketplace/typo-ci) app would need to be installed and enabled for this repo.

File: .typo-ci.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+# en
+# en_GB
+# es
+# fr
+# it
+# pt
+# pt_BR
+dictionaries:
+    - en
+    - en_GB
```
Comment (akelad) [inline .typo-ci.yml:16]: are we sure it should be both en and en_GB? should probably just be one of the two across the docs

**Comment by akelad** [comment]
@tmbo could you add the typoci app to our organisation please?

File: .typo-ci.yml (hunk 1 above)
Comment (melindaloubser1) [reply .typo-ci.yml:16]: that's fair I'll switch it on the test pr and see what happens

**Comment by tmbo** [comment]
@akelad @melindaloubser1 just added typoci to the repo ✔️

File: .typo-ci.yml (hunk 1 above)
Comment (melindaloubser1) [reply .typo-ci.yml]: I guess we should be consistent with our blog? So US english. Quite a few british spelling changes then need to happen, which is fine if everyone realizes that

**Pull request merged**